src/test/resources/regressions/features/loops/loops-break1.gobra

```diff
@@ -0,0 +1,10 @@
+// Any copyright is dedicated to the Public Domain.
+// http://creativecommons.org/publicdomain/zero/1.0/
+
+package pkg
+
+func foo() {
+	for true {
+		break
+	}
```

Comment:
I would add an `assert false` after the loop to show that this loop indeed terminates (and thus `break` behaves as expected)